src/components/personalia-header/PersonaliaHeader.tsx

```diff
@@ -32,6 +33,7 @@ const PersonaliaHeader = ({ personalia }: PersonaliaHeaderProps) => {
                     Søker er skjermet
                 </Tag>
             )}
+            <HarBarnFortroligAdresse barn={personalia.barn} />
```

Comment:
Ser bra ut! Men har et forslag: skulle vi lagd en klasse ut av Personopplysninger (nå et interface), og stappe inn en egen funksjon der for å hente en liste med barn som er fortrolig/strengtFortrolig? Da knytter vi den sjekken til Personopplysninger-dataen, istedenfor at det ligger i React-komponenten.

Reply:
Enig, da fikser jeg det.
- [ ] Flytt sjekk om barn er fortrolig til Personopplysninger